package.json

```diff
@@ -24,9 +24,11 @@
   "author": "Christian Theilemann <christian@solvvy.com>",
   "license": "MIT",
   "dependencies": {
-    "@google-cloud/dlp": "~0.11.0",
     "lodash": "^4.17.11"
   },
+  "peerDependencies": {
+    "@google-cloud/dlp": "~0.11.0"
+  },
   "devDependencies": {
```

Comment:
simply add @google-cloud/dlp also to devDependencies. This should make the tests pass even without modifying the github workflow.

Reply:
@geekflyer great point, thank you for pointing this out. I have added `@google-cloud/dlp` to devDependencies.